tools/build_config/Jenkinsfile

```diff
@@ -281,13 +281,18 @@ pipeline {
             }
 
           } else {
+            if (env.PR_LABELS && "${env.PR_LABELS}" != "[]") {
```

Comment:
The second clause in unnecessary; an empty list of labels is as important as a populated one for logging. Since the default (empty) case gives a non-empty string that distinguishes it from the non-labels-passed (i.e non-PR) pipeline case trapped by the first clause.